### ISSUE TITLE
Bot ignore Azure pipelines failures

### ISSUE THREAD

**Issue by Geod24** [dlang/dlang-bot#250]
Seen in https://github.com/dlang/dmd/pull/11286
It broke HEAD a few weeks ago.

I'm going to go out of a hunch and guess that's because Azure pipeline are Github checks. A way to confirm this theory would be to have an always-failing Github CI check and see if the bot merges.

**Comment by wilzbach** [comment]
Yes, dlang-bot is a lot older than the GitHub checks, so yep it isn't even aware of them.

**Comment by CyberShadow** [comment]
Is this a dupe of #243?
